demo/config.json

```diff
@@ -8,6 +8,8 @@
 	"PowDepth": 0,
 	"RpcPort": 8401,
 	"RpcHost": "0.0.0.0",
+        "KeyFile": "/etc/letsencrypt/live/demo.erdstall.dev/privkey.pem",
```

Comment:
I think it would be cleaner to create a certificate for the `operator` instead of copying it from the `demo` server.

Reply:
I actually think we shouldn't set TLS paths in the `demo/config` file, but document the json fields. This is a path not relevant for this public repo.